var/spack/repos/builtin/packages/py-hatchet/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyHatchet(PythonPackage):
+    """Hatchet is an analysis tool for structured tree or graph performance data
+    using an indexed Pandas dataframe."""
+
+    homepage = "https://github.com/LLNL/hatchet"
+    url      = "https://github.com/LLNL/hatchet/archive/v1.0.0.tar.gz"
+
```

Comment:
Is anyone from LLNL interested in being a "maintainer" for this package?

Reply:
Yes, I'll add that field here to list maintainers.